### ISSUE TITLE
Target platform

### ISSUE THREAD

**Issue by xixixao** [leporello-js/leporello-js#1]
Have you considered targeting VS Code instead of the browser?

I know that the editor is limited in presentation options, but maybe monaco could be forked.

Curious to hear more about how this project can become viable/complete.

**Comment by dmitry-vsl** [comment]
Building VS Code plugin is my top priority. I am planning to integrate Leporello.js with built-in ts-server, so code would be parsed and analyze once by ts-server, and Leporello plugin would reuse syntax tree from ts-server.

However, it would require several months of development effort.

**Comment by xixixao** [comment]
Yeah, this is gonna be a ton of work but definitely feels like a more viable path!

Everything here is just incredible work. You could probably sign up for YC with this project.
